libs/elements/layout/src/lib/components/elewa-hero/elewa-hero.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'elewa-group-elewa-hero',
+  templateUrl: './elewa-hero.component.html',
+  styleUrls: ['./elewa-hero.component.scss'],
+})
+export class ElewaHeroComponent {
+  @Input() title = 'HOW TO RECOGNIZE AN ELEWA MEMBER';
+  @Input() subTitle = 'Dependable, Creative, Supportive, Open Minded and Fun*';
+  @Input()  paragraph = '*SERIOUS WHEN IT MATTERS';
+
```

Comment:
You've not added the optional text at the bottom of the hero.
should have an *ngIf directive